packages/components/pages/f-account-info/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-account-info",
   "description": "Fozzie Account Info - The account information page",
-  "version": "0.5.0",
+  "version": "0.7.0",
```

Comment:
What happened to `0.6.0`?

Reply:
Nobody talks about `0.6.0,` now you've mentioned it #awks

Reply:
`0.6.0` was in progress and now closed - https://github.com/justeat/fozzie-components/pull/1520